src/frontend/src/pages/DesignReviewSummaryModal.tsx

```diff
@@ -0,0 +1,285 @@
+            <Grid item sx={{ display: 'flex', alignItems: 'start', marginTop: '7px' }}>
+              <Typography>Required: </Typography>
+            </Grid>
+            <Grid item xs={10}>
+              <Grid container>
+                {requiredMembers.map((member, index) => (
+                  <Grid item key={index}>
+                    <DesignReviewMemberPill
+                      user={member}
+                      handleClick={() => {
+                        handleRemoveRequiredMember(member);
+                      }}
+                    />
+                  </Grid>
+                ))}
+              </Grid>
+            </Grid>
+          </Grid>
+          <Grid item xs={12}>
+            <Grid container>
+              <Grid item sx={{ display: 'flex', alignItems: 'start', marginTop: '7px' }}>
+                <Typography>Optional: </Typography>
+              </Grid>
+              <Grid item>
+                <Grid container>
+                  {optionalMembers.map((member, index) => (
+                    <Grid item key={index}>
+                      <DesignReviewMemberPill
+                        user={member}
+                        handleClick={() => {
+                          handleRemoveOptionalMember(member);
+                        }}
+                      />
+                    </Grid>
+                  ))}
+                </Grid>
+              </Grid>
+            </Grid>
+          </Grid>
+        </Grid>
+      </Grid>
+    </Box>
+  );
+
+  const SummaryDetails: React.FC = () => (
+    <Grid container direction="row" alignItems="center" justifyContent="center" columnSpacing={1}>
+      <Grid item xs={3}>
+        <DesignReviewPill
+          icon={<AccessTimeIcon />}
+          isLink={false}
+          displayText={`${meetingDatePipe(designReview.dateCreated)} ${availabilityStartTimePipe(
+            designReview.meetingTimes
+          )}`}
+        />
+      </Grid>
+      <Grid item xs={3}>
+        {editing ? (
+          <TextField
+            variant="standard"
+            onChange={(event: ChangeEvent<HTMLInputElement>) => {
+              setLocationState(event.target.value);
+            }}
+            value={locationState}
+            label="Enter Location"
+          ></TextField>
+        ) : (
+          <DesignReviewPill icon={<LocationOnIcon />} isLink={false} displayText={locationState ?? 'Online'} />
+        )}
+      </Grid>
+      <Grid item xs={3}>
+        {editing ? (
+          <TextField
+            variant="standard"
+            onChange={(event: ChangeEvent<HTMLInputElement>) => {
+              setDocumentState(event.target.value);
+            }}
+            value={documentState}
+            label="Enter Docs URL"
+          ></TextField>
+        ) : (
+          <DesignReviewPill
+            isLink
+            icon={<DescriptionIcon />}
+            linkURL={documentState ?? ''}
+            displayText={documentState ? 'Questions Doc' : 'No Doc'}
```

Comment:
make it so that this doesn't go onto two lines (either rename or respace the items a bit)